docs/Configuration-Settings.md

```diff
@@ -180,11 +180,26 @@ number.
     specified for `tachyon.master.hostname`.</td>
 </tr>
 <tr>
+  <td>tachyon.master.bind.host</td>
+  <td>0.0.0.0</td>
+  <td>The hostname Tachyon's master node binds to.</td>
```

Comment:
If the machine has two external network interfaces and one loopback interface, user might only want to bind the service on two external interfaces. In such case, the "*.bind.host" could have more possible values. However, to simplify Tachyon setup, we can limit the bind values as either "*.hostname" or "0.0.0.0" for now.